src/components/InputWithLabel.js

```diff
@@ -26,5 +26,8 @@ return(
 
 };
 
+InputWithLabel.propTypes={
```

Comment:
your props for this component are `handleTitleChange, todoTitle, children` and it are the same items you expect to see proptype checked: 
handleTitleChange: PropTypes.func, 
.....

I'd suggest to rename `children` to 'label' - this name is describes better the purpose if this prop